test/python/docker-compose.yml

```diff
@@ -6,10 +6,11 @@ version: '3'
 services:
   ms0:
     container_name: "ms0"
-    image: rust:latest
+    image: shubham1494/lvm-test:latest
```

Comment:
don't think this is what we want, do we?

Reply:
Once the dockerfile is approved we can have this image in openebs docker hub

Reply:
We don't need to have docker files at all for the tests.